samples/apps/copilot-chat-app/webapi/DeploymentTemplates/README.md

```diff
@@ -1,11 +1,13 @@
 # Deploying Semantic Kernel to Azure in a web app service
 
+This document details how to deploy Semantic Kernel as a backend service that can be used by other applications or by a frontend such as the one for [Copilot Chat](../../webapp/README.md).
```

Comment:
(nit) Should we add a note here that this is to deploy the infrastructure? Or be specific that re-run the deployment script will create new set of infra every time (if my understanding is correct...)?